pallets/grants/src/weights.rs

```diff
@@ -19,85 +19,44 @@
-	}
-	fn claim() -> Weight {
-		(50_695_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(7 as Weight))
-			.saturating_add(T::DbWeight::get().writes(4 as Weight))
-	}
-	fn cancel_all_vesting_schedules() -> Weight {
-		(128_611_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(8 as Weight))
-			.saturating_add(T::DbWeight::get().writes(6 as Weight))
-	}
-	fn overwrite_vesting_schedules() -> Weight {
-		(40_916_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(6 as Weight))
-			.saturating_add(T::DbWeight::get().writes(5 as Weight))
-	}
-}
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {	fn add_vesting_schedule() -> Weight {
+		(103_704_000 as Weight)			.saturating_add(T::DbWeight::get().reads(9 as Weight))			.saturating_add(T::DbWeight::get().writes(6 as Weight))	}	fn claim() -> Weight {
+		(62_420_000 as Weight)			.saturating_add(T::DbWeight::get().reads(8 as Weight))			.saturating_add(T::DbWeight::get().writes(4 as Weight))	}	fn cancel_all_vesting_schedules() -> Weight {
+		(140_385_000 as Weight)			.saturating_add(T::DbWeight::get().reads(9 as Weight))			.saturating_add(T::DbWeight::get().writes(6 as Weight))	}	fn overwrite_vesting_schedules() -> Weight {
+		(47_563_000 as Weight)			.saturating_add(T::DbWeight::get().reads(7 as Weight))			.saturating_add(T::DbWeight::get().writes(5 as Weight))	}}
 
 // For backwards compatibility and tests
-impl WeightInfo for () {
-	fn add_vesting_schedule() -> Weight {
-		(90_319_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(8 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(6 as Weight))
-	}
-	fn claim() -> Weight {
-		(50_695_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(7 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(4 as Weight))
-	}
-	fn cancel_all_vesting_schedules() -> Weight {
-		(128_611_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(8 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(6 as Weight))
-	}
-	fn overwrite_vesting_schedules() -> Weight {
-		(40_916_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(5 as Weight))
-	}
-}
+impl WeightInfo for () {	fn add_vesting_schedule() -> Weight {
+		(103_704_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(9 as Weight))			.saturating_add(RocksDbWeight::get().writes(6 as Weight))	}	fn claim() -> Weight {
+		(62_420_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(8 as Weight))			.saturating_add(RocksDbWeight::get().writes(4 as Weight))	}	fn cancel_all_vesting_schedules() -> Weight {
+		(140_385_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(9 as Weight))			.saturating_add(RocksDbWeight::get().writes(6 as Weight))	}	fn overwrite_vesting_schedules() -> Weight {
+		(47_563_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(7 as Weight))			.saturating_add(RocksDbWeight::get().writes(5 as Weight))	}}
```

Comment:
The formatting looks to be messed up

Reply:
Could be an issue with the template file too. For now, run `cargo +nightly fmt` after a rebase